Source/Core/Core/IOS/Network/SSL.cpp

```diff
@@ -330,10 +337,14 @@ IPCCommandResult NetSSL::IOCtlV(const IOCtlVRequest& request)
     {
       WII_SSL* ssl = &_SSL[sslID];
       const std::string cert_base_path = File::GetUserPath(D_SESSION_WIIROOT_IDX);
-      const std::vector<u8> client_cert =
-          ReadCertFile(cert_base_path + "/clientca.pem", s_client_cert_hash);
-      const std::vector<u8> client_key =
-          ReadCertFile(cert_base_path + "/clientcakey.pem", s_client_key_hash);
+      const std::vector<u8> client_cert = ReadCertFile(
+          cert_base_path + "/clientca.pem", s_client_cert_hash, m_built_in_cert_error_shown);
+      const std::vector<u8> client_key = ReadCertFile(
+          cert_base_path + "/clientcakey.pem", s_client_key_hash, m_built_in_cert_error_shown);
+      // If any of the required files fail to load, show a panic alert, but only once
+      // per IOS instance (usually once per emulation session).
+      if (client_cert.empty() || client_key.empty())
+        m_built_in_cert_error_shown = true;
```

Comment:
Nitpick, maybe we can shorten this var down to `m_cert_error_shown`. Var name is needlessly verbose as is. Feel free to disregard if you feel differently.

Reply:
Funnily enough, that's the name I went with at first, but I added "built-in" in an attempt to make it clear that the error would be about built-in cert data failing to load, and not cert verification errors.

After reading it again, it looks like it doesn't actually clarify things that much, and makes the name a bit too long :( I'll change it back to `m_cert_error_shown`.

Reply:
Changed